crates/astria-sequencer/src/app/storage/values/block_height.rs

```diff
@@ -0,0 +1,55 @@
+use std::fmt::{
+    self,
+    Display,
+    Formatter,
+};
+
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+
+use super::{
+    Value,
+    ValueImpl,
+};
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(in crate::app) struct BlockHeight(u64);
```

Comment:
Should probably have a domain type for this.

Reply:
Ticketed at #1604.